aboutlibraries-compose/src/main/kotlin/com/mikepenz/aboutlibraries/ui/compose/Libraries.kt

```diff
@@ -0,0 +1,218 @@
+import androidx.compose.foundation.clickable
```

Comment:
Any particular reason why the package is missing from this file?

Without the package the import of the Composable looks a little weird:

```kotlin
import LibrariesContainer
import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.compose.runtime.Composable
import androidx.compose.runtime.LaunchedEffect
import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
import androidx.navigation.NavHostController
import androidx.navigation.compose.NavHost
import androidx.navigation.compose.rememberNavController
```

Instead of `import LibrariesContainer` I would expect to have `import com.mikepenz.aboutlibraries.ui.compose.LibrariesContainer`

Reply:
@leinardi oh that was an oversight, will have a look to fix that. thank you very much. 

10.x is still very early, so all feedback is highly appreciated :)